ArduCopter/mode_autotune.cpp

```diff
@@ -47,6 +47,7 @@
 #define AUTOTUNE_LEVEL_RATE_RP_CD          1000     // rate which qualifies as level for roll and pitch
 #define AUTOTUNE_LEVEL_RATE_Y_CD            750     // rate which qualifies as level for yaw
 #define AUTOTUNE_REQUIRED_LEVEL_TIME_MS     500     // time we require the copter to be level
+#define AUTOTUNE_LEVEL_TIMEOUT_MS          2000     // time we require the copter to be level
```

Comment:
Using the same comment isn't very good, something like `time after which we consider the copter to be level` would be better I think.